src/pywink/devices/standard.py

```diff
@@ -289,6 +304,9 @@ def _update_state_from_response(self, response_json):
                 outlet['last_reading']['connection'] = power_strip_reading.get('connection')
                 self.json_state = outlet
 
+    def pubnub_update(self, json_response):
```

Comment:
Really, I think it could probably get moved into base and then get overridden. The only one that is required is the powerstrip. This is because the json needs filtered through the _update_state_from_response to get the individual outlet out. Wouldn't want the app to call that or have to handle extracting just the outlets json to set json_state.